cmd/main.go

```diff
@@ -34,8 +33,12 @@ func Run(name string, args []string) error {
 
 	switch {
 	case opts.version:
-		fmt.Fprintf(os.Stdout, "gotestsum version %s\n", version)
-		return nil
+		info, ok := debug.ReadBuildInfo()
+		if !ok {
+			return fmt.Errorf("failed to read version info")
+		} else {
+			fmt.Fprintf(os.Stdout, "gotestsum version %s\n", info.Main.Version)
+		}
```

Comment:
There is no need for an else statement when you return

```suggestion
		if !ok {
			return fmt.Errorf("failed to read version info")
		}

		fmt.Fprintf(os.Stdout, "gotestsum version %s\n", info.Main.Version)
```

Reply:
Updated. Thank you for the suggestion.

Reply:
Thanks for updating the PR.

Bow, let's wait for the feedbacks of a maintainer